tools/bgpreader.c

```diff
@@ -71,6 +72,35 @@
   "# <elem-type>: R RIB, A announcement, W withdrawal, S state message\n"      \
   "#\n"                                                                        \
 
+#define LONG_OPTIONS                                                           \
+  (struct option [])                                                           \
+  {                                                                            \
+    {"filterstring",    required_argument, 0, 'f'},                            \
+    {"interval",        required_argument, 0, 'I'},                            \
+    {"interface",       required_argument, 0, 'd'},                            \
+    {"option",          required_argument, 0, 'o'},                            \
```

Comment:
I would personally prefer ```data-interface-option```. It would be consistent to ```data-interface``` and the other options, which do not contain any abbreviations.